org-formation/040-budgets/cur.yaml

```diff
@@ -0,0 +1,42 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'Monthly Cost and Usage Report'
+Parameters:
+  resourcePrefix:
+    Type: String
+Resources:
+  CostReportBucket:
+    Type: "AWS::S3::Bucket"
+    Properties:
+      BucketName: !Sub "${resourcePrefix}-cost-reports"
+      AccessControl: PublicRead
```

Comment:
hmm, i think `public read` will allow anybody access to the bucket.  Do we really want that?

Reply:
I'm on the fence about this, do we have a need to keep the information confidential?

Reply:
to clarify, i believe this allows anybody on the internet access to the cost reports.